static/js/components/ShareTooltip.js

```diff
@@ -37,11 +38,13 @@ export class ShareTooltipHelper extends React.Component<HelperProps> {
   }
 
   render() {
-    const { url, hideSocialButtons } = this.props
+    const { url, hideSocialButtons, objectType } = this.props
 
     return (
       <div className="tooltip">
-        <div className="tooltip-text">Share a link to this post:</div>
+        <div className="tooltip-text">
+          {`Share a link to this ${objectType || "post"}:`}
+        </div>
         <input ref={this.input} readOnly value={url} />
         <div className="bottom-row">
           {hideSocialButtons ? null : (
```

Comment:
`hideSocialButtons` is returning `False` for private user lists / learning paths

Reply:
ah, forgot about that option, I'll add a check for it

Reply:
ah, forgot about that option, I'll add a check for it